### PR TITLE
Order feeds from most recent to oldest

### PR DESCRIPTION
**Pull request by jpmartinspt**
# Done
Show most recent entries first in feeds endpoints.

# QA
- In `www.ubuntu.com` replace the `canonicalwebteam.blog` dependency with this branch.
- Go to feeds endpoints and make sure the most recent entries come first.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # IMPORTANT: this file is autogenerated. Do not edit it manually.
+Flask (>=1.0,<2.0)
```
Comment (nottrobin) [inline requirements.txt]: Double flask?

File: setup.py [hunk 2]
```diff
@@ -46,6 +46,8 @@
     packages=find_packages(),  # Required
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
+        'django[django] (>=2.2,<3.0)',
```
Comment (nottrobin) [inline setup.py]: This is a funny dependency

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
 Flask (>=1.0.1,<2.0.0)
 canonicalwebteam.http (>=1.0.1,<2.0.0)
 django (>=2.2,<3.0)
+django[django] (>=2.2,<3.0)
```
Comment (nottrobin) [inline requirements.txt]: Double Django?

File: setup.py [hunk 4]
```diff
@@ -46,6 +46,8 @@
...
+        'Flask (>=1.0,<2.0)',
```
Comment (nottrobin) [inline setup.py]: These should be under an `extras_require` section

File: pyproject.toml [hunk 5]
```diff
@@ -13,16 +13,14 @@ license = "LGPL-3.0"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-django = {version = "^2.2", extras = ["django"]}
+django = {version = "^2.2", extra = ["django"]}
```
Comment (nottrobin) [inline pyproject.toml]: Actually, I think `extras` is the right word. And it should be `extras` for Flask too.

Comment (nottrobin) [reply pyproject.toml]: At least that's what the example here suggests: https://github.com/sdispater/poetry#introduction

Comment (jpmartinspt) [reply pyproject.toml]: `poetry install` fails with `extras`.

Comment (nottrobin) [reply pyproject.toml]: yeah that's cos it's right =) what we should actually be doing is `optional = True` or something I think

Comment (nottrobin) [reply pyproject.toml]: https://poetry.eustace.io/docs/pyproject/#extras

Comment (nottrobin) [reply pyproject.toml]: But the simplest thing to do would just be to purely update the version in these files for the time being. None of this stuff is related to this PR.

Comment (jpmartinspt) [reply pyproject.toml]: Yup, but that then fails for `poetry-setup` :man_shrugging:

Comment (nottrobin) [reply pyproject.toml]: But FWIW, I think the right syntax should be:

``` toml
django = {version = "^2.2", optional = true}
flask = {version = "^1.0", optional = true}

[tool.poetry.extras]
django = ["django
flask = ["flask"]
```

Comment (nottrobin) [reply pyproject.toml]: Yeah but you don't *need* to fix or run `poetry-setup` for this PR do you? You can literally just update the number in `setup.py` and `pyproject.toml`

Comment (jpmartinspt) [reply pyproject.toml]: Yeah, done that now.

**Comment by nottrobin** [comment]
The diff now looks good, but it also looks like you're trying to overwrite history in this PR:

image

Could you separate your work out into a new commit?

**Comment by jpmartinspt** [comment]
Fixed now.

**Pull request merged**